lib/Service/RecipeService.php

```diff
@@ -748,7 +752,7 @@ public function addRecipe($json) {
 			$thumb_image = new Image();
 			$thumb_image->loadFromData($full_image_data);
 			$thumb_image->fixOrientation();
-			$thumb_image->resize(128);
+			$thumb_image->resize(256);
```

Comment:
This will default the thumb image size to 256x256. We should make sure, this is no problem as the old files are still 128x128.

Reply:
I would like to put this out for discussion. Actually this change doesn’t make too much sense, since currently the images are shown in the lists in 105px by 105px - so 128 is more sensible in the sense that is less data to be transfered. However, there seems to be the need for larger thumbnails (#406). We could also keep the 128px thumbnail `thumb.jpg` and create a 256x256 version as `thumb@2x.jpg`

Reply:
I do not consider this too much of a problem to use 256x256. It is only a few kB per recipe.